src/sagas/wallet.js

```diff
@@ -291,16 +279,7 @@ export function* loadTokens() {
 
   const wallet = yield select((state) => state.wallet);
 
-  const registeredTokens = call(async () => {
-    const tokens = [];
-    for await (const token of wallet.storage.getRegisteredTokens()) {
```

Comment:
`wallet.storage.getRegisteredTokens` exposes an `asyncIterator` method so the runtime can iterate properly, but redux-saga only allows for iterations on objects with an `iterator` method. The async generator is not identified as an iterable and the for syntax will fail.

This was solved by the `getRegisteredTokens` helper which iterates manually so it does not rely on the runtime lib.